domains/fa-fifi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fa-fifi",
+        "email": "fafifi1997@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=aybl6pibwuv4qzq9kxmrjam9vfcq0iunrbx_aexpfve"
```

Comment:
TXT record alone doesn't do anything.

Reply:
> TXT record alone doesn't do anything.

Yup, you're right but since CNAME record cannot coexist with other records, I decided to put TXT record first only for verification on Google Search Console. Once it is verified, I will remove it and add a CNAME record to my github.io page. @DEV-DIBSTER Hope you understand and if there is any other way to achieve that, I open to a discussion.